README.md

```diff
@@ -682,6 +683,19 @@ Example:
 ./pkb.py --cloud=GCP --zone=us-central1-a --benchmarks=iperf,ping
 ```
 
+The disk type names vary by provider, but the following table summarizes some
+useful ones.
+
+Cloud name | Network-attached SSD | Network-attached HDD
```

Comment:
Can you update for @meteorfox  comment ... Please add Rackspace.  Thanks!

boot - uses the boot disk
local - uses local disks that is not the boot disk
cbs-sata - remotely attached HDD volume from Cloud Block Storage
cbs-ssd - remotely attached SSD volume from Cloud Block Storage